src/hayai_clock.hpp

```diff
@@ -56,6 +56,10 @@
 // Unix
 #elif defined(__unix__) || defined(__unix) || defined(unix)
 
+#if defined (__unix__) || (defined (__APPLE__) && defined (__MACH__))
```

Comment:
For good measure, we should lift this to the top level. Otherwise, the `__APPLE__` component is pointless. Let's also extend it to be `defined(__unix__) || defined(__unix) || defined(unix) || (defined (__APPLE__) && defined (__MACH__))` to ensure complete logical overlap.